src/mmrelay/db_utils.py

```
@@ -756,24 +973,252 @@ def _update_names_core(
+            a Meshtastic ID when the normalized name is absent/empty.
+        delete_stale_names (Callable[[set[str]], int]): Function used to delete
+            rows whose Meshtastic IDs are absent from the snapshot.
+
+    Returns:
+        bool: True when all save operations succeeded; False if any save failed.
+    """
+    if not nodes:
+        return True
+
+    if name_key == NODE_NAME_FIELD_LONG:
+        get_name = lambda entry: entry.long_name
```

Ruff (0.15.6)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `get_name` as a `def`

(E731)